src/libs/assignRef.ts

```diff
@@ -0,0 +1,19 @@
+import {MutableRefObject, RefCallback} from 'react';
```

Comment:
I guess this file path is okey... I would personally prefer `refUtils.ts` with exported `assignRef`, with the possibility of adding more ref-related utils in the future, **but** it would cause a linter error if I remember correctly... Expensify likes default exports, I don't.

Reply:
Definitely, but as you said it would require to import all of the exports at once or change their type (from default to named ones). 

Let's keep it as-is 👍